### DISCUSSION TITLE
Deployment of New Master Node using 2.3.52 "Will it work"

### DISCUSSION THREAD

**Discussion by jdpadro** [Security-Onion-Solutions/securityonion#4448] [Unsupported Versions] [answered]
OK so this evening I went back to the Virtual drawing board and decided to give it another go. I created the sensor and attempted the connection to the manager and it worked so the Tar ball was identified correctly and the installation proceeded as expected. The install on the sensor completed and the sensor was properly added to the Grid. 

Thanks

**Comment by TOoSmOotH** [comment, marked as answer]
Glad it worked

**Reply by jdpadro** [reply]
Yep so am I. For my production environment I reinsstalled the manager and the first sensor and since it worked I started moving my other sensors onto 2.3.52. Since I should have all my local and remote sensors added to the new environment.
